bin/lsif-php

```diff
@@ -36,11 +36,26 @@ if (\ini_set('memory_limit', $memoryLimit) === false) {
     exit(1);
 }
 
+$id = intval($options['id'] ?? '0');
+if ($id < 0) {
+    echo "Invalid id.\n";
+    exit(1);
+}
+
+$filename = $options['filename'] ?? 'dump.lsif';
+if (!\is_string($filename) || \substr($filename, -5) !== '.lsif') {
+    echo "Invalid filename. Must be a file ending with .lsif\n";
```

Comment:
Could you add the new options the help text (inside the `if` statement at line 19)?